### ISSUE TITLE
How to use  momentum in the learning rate

### ISSUE THREAD

**Issue by gaowenxin95** [dmlc/xgboost#6089]
When I adjust the learning rate parameter, the learning rate remains constant during the training process，so，l would like to know how to get an adaptive learning rate，for example,using  momentum or sth?

@kashif @mattn @darxriggs @smly @omo @JiaxiangBU

**Comment by trivialfis** [comment]
There's call back function.
